DataFormats/HcalRecHit/interface/HFPreRecHit.h

```diff
@@ -28,6 +30,29 @@ class HFPreRecHit
     // index is out of range.
     const HFQIE10Info* getHFQIE10Info(unsigned index) const;
 
+    // Quantities simply added from both anodes
+    float charge() const;
+    float energy() const;
```

Comment:
will this remain "simply added" ?
Initially, PreRecHit looked like a slightly wrapped access to 1 or 2 HFQIE10Info.
Then, all proper combinations into a single observable would be in HFRecHit.

Maybe make it chargeTotal and energyTotal? 

.. it's minimal enough for now to stay as is, but it seems to me like new functionality is creeping in